File: baekjoon_all/02000+/boj_2753.py
# Solve 2022-05-16
# Update 2023-02-22 #

year = int(input())

if year % 400 == 0:
    print(1)
elif year % 100 == 0:
    print(0)
elif year % 4 == 0:
    print(1)
else:
    print(0)

File: baekjoon_all/02000+/boj_2753.cpp
// Solve 2021-12-18
// Update 2023-08-02

#include <iostream>
using namespace std;

int main() {
    int year;
    cin >> year;

    if (year % 400 == 0 || year % 4 == 0 && year % 100 > 0) {
        cout << 1 << '\n';
    }
    else {
        cout << 0 << '\n';
    }

    return 0;
}
